src/utils/components/PendingChanges/utils/constants.ts

```diff
@@ -39,6 +40,8 @@ export enum VirtualMachineDetailsTabLabel {
   Snapshots = 'Snapshots',
   // t('Scripts')
   Scripts = 'Scripts',
-  // t('Diagnostic)
+  // t('Diagnostic')
   Diagnostic = 'Diagnostic',
+  // t('Configurations')
+  Configurations = 'Configurations',
```

Comment:
I believe this should be 'Configuration'.

Reply:
Well in the design its Configurations but you know English, lol. will change